onnx/defs/schema.h

```diff
@@ -20,6 +21,7 @@
 #include "data_type_utils.h"
 #include "onnx/common/constants.h"
 #include "onnx/defs/shape_inference.h"
+#include "onnx/onnx-operators_pb.h"
```

Comment:
I'm seeing we're also having onnx-operators-ml.proto.  You may either remove this proto to have only one proto file, or handle both cases (ONNX_ML=0 and ONNX_ML=1).

Reply:
This file is actually a auto-switch between onnx-operators-ml.pb.h and onnx-operators.pb.h